core/src/main/java/com/yahoo/ycsb/generator/FileGenerator.java

```diff
@@ -73,16 +73,12 @@ public String lastString()
 	/**
 	 * Reopen the file to reuse values.
 	 */
-	public synchronized void reloadFile()
+	public synchronized void reloadFile() throws IOException
```

Comment:
I think the reason was just bad code. It doesn't make sense for this method to complete normally in case of an exception. Same comment for nextValue() too. I'll change it to close the reader as a try with resource statement.